engine/src/main/java/io/camunda/zeebe/engine/state/migration/DbMigrationState.java

```diff
@@ -387,6 +422,14 @@ public void migrateProcessDefinitionVersions() {
 
   @Override
   public void migrateProcessStateForMultiTenancy() {
+    deprecatedProcessCacheColumnFamily.forEach(
+        (key, value) -> {
+          value.setTenantId(TenantOwned.DEFAULT_TENANT_IDENTIFIER);
```

Comment:
🤔 We don't even need this anymore. I know I wrote this down some time ago, but that was back when we had the records set the tenantId to an empty string as default value.

Reply:
Resolved in a later commit

Reply:
❓ Could it hurt us? Let's say that someone enabled multi-tenancy, and we run these migrations, then we would not encounter entries in deprecatedProcessCache. So, I think we're fine just keeping it. I don't believe it comes with a performance hit, we'll to migrate the entries anyways